packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -210,6 +245,10 @@ export function ArticleMetadataPanel({value, onClose, onChange}: ArticleMetadata
                   onChange={seoTitle => onChange?.({...value, seoTitle})}
                 />
                 <HelpBlock>{t('articleEditor.panels.seoTitleHelpBlock')}</HelpBlock>
+                <label style={{textAlign: 'right'}}> {`${t("articleEditor.panels.charCount")} ${value.seoTitle.length}`} </label>
```

Comment:
I guess we can have it like it's described here: https://www.i18next.com/translation-function/interpolation
to make use of the variable you added in the locale file ;-)

Reply:
I actually removed this part (${t("articleEditor.panels.charCount")}) as we only need to view the number without a text beside it.

Reply:
I think it will be helpful? @HanaAyman  @roosnic1 what do you think?

Reply:
@bishoymelek it think it's pretty clear what it is and we can omit the text.